.github/docker/vault/entrypoint/container.sh

```
@@ -0,0 +1,44 @@
+log_level     = "info"
+EOM
+
+vault server \
+  -config="/vault/config" \
+  -dev-tls \
+  -non-interactive \
+  -tls-skip-verify \
+  -dev-listen-address="0.0.0.0:8200" &
+
+# wait vault server is completely started
+for i in $(seq 1 30); do grep -q 'successful mount' /vault/logs/vault.log && break || sleep 1; done
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)